src/js/components/FieldGroup/FieldGroup.js

```diff
@@ -0,0 +1,56 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import Field from '../Field'
+import FieldLabel from '../FieldLabel'
+import FieldMessage from '../FieldMessage'
+import Input from '../Input'
+import TextArea from '../TextArea'
+import Checkbox from '../Checkbox'
+import Radio from '../Radio'
+import Toggle from '../Toggle'
+import Select from '../Select'
+
+class FieldGroup extends PureComponent {
+  renderFieldComponent = () => {
+    const { name, field, fieldProps } = this.props
+
+    let FieldTag = Input
+    if (field === 'select') { FieldTag = Select }
+    else if (field === 'checkbox') { FieldTag = Checkbox }
+    else if (field === 'radio') { FieldTag = Radio }
+    else if (field === 'toggle') { FieldTag = Toggle }
+    else if (field === 'textarea') { FieldTag = TextArea }
```

Comment:
Feels like this could be a map
```
const FieldTypes = {
  select: Select,
  checkbox: CheckBox,
  radio: Radio,
  toggle: Toggle,
  textarea: TextArea
}
const FieldTag = (FieldTypes[field] || Input)
```
Makes is super easy to add field types

Reply:
I'm not opposed to a change like this.  I don't really see this changing as there are not a ton of 'input' field types.  But it seems to be a nice way to clean it up.  👍